components/scream/src/share/property_checks/field_nan_check.cpp

```diff
@@ -87,20 +96,22 @@ bool FieldNaNCheck::check_impl(const Field& field) const {
       }
       break;
     default:
-      EKAT_ERROR_MSG ("Error! Unsupported field rank.\n");
+      EKAT_ERROR_MSG ("Error! Unsupported f rank.\n");
```

Comment:
Could say `field` instead of `f`. Also, why not print the field's name and its (unsupported) rank?

Reply:
Ah, yeah. I did a find+replace earlier, but accidentally edited also comments. :)